examples/gallery/histograms/blockm.py

```diff
@@ -41,8 +41,8 @@
 
 fig.shift_origin(xshift="w+5c")
 
-# Calculate number of total locations within 150x150 minute bins via
-# blockmean's summary parameter
+# Calculate number of total locations within 150x150 arc-minute bins
+# via blockmean's summary parameter
```

Comment:
```suggestion
# Calculate number of total locations within 150x150 arc-minute bins
# with blockmean's summary parameter
```

I think using "with" makes more sense than "via".

Reply:
I can not say reliable what is here correct or better, but changed it :wink:.